src/darwin/Framework/CHIP/MTRCommissioningParameters.h

```diff
@@ -122,7 +122,34 @@ MTR_AVAILABLE(ios(16.1), macos(13.0), watchos(9.1), tvos(16.1))
  * The FeatureMap attribute of all Network Commissioning clusters on the commissionee
  * will always be read and does not need to be included in this list.
  */
-@property (nonatomic, copy, nullable) NSArray<MTRAttributeRequestPath *> * extraAttributesToRead MTR_PROVISIONALLY_AVAILABLE;
+@property (nonatomic, copy, nullable) NSArray<MTRAttributeRequestPath *> * extraAttributesToRead MTR_AVAILABLE(ios(26.2), macos(26.2), watchos(26.2), tvos(26.2));
+
+/**
+ * Whether to force a network scan before requesting Wi-Fi credentials.
+ * The default is NO.
+ *
+ * Even if this value is NO a scan may still be performed.
+ *
+ * This value will be ignored if Wi-Fi credentials are provided or not needed.
+ *
+ * NOTE: Not all APIs that take MTRCommissioningParameters pay attention to this
+ * flag.
+ */
+@property (nonatomic, assign) BOOL forceWiFiScan MTR_AVAILABLE(ios(26.2), macos(26.2), watchos(26.2), tvos(26.2));
```

Comment:
I still need to test that this works properly.  Leaving this comment to prevent merge until I do that.